tools/StaticAnalysis/Exceptions/Az.HDInsight/BreakingChangeIssues.csv

```diff
@@ -0,0 +1,5 @@
+"AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
```

Comment:
These lines will suppress breaking change warning of static analysis, which are not expected in a breaking change notice release.

Reply:
Hi @msJinLei ,  I want to call out breaking change and remove outdated alias in the same PR.

Reply:
> Hi @msJinLei , I want to call out breaking change and remove outdated alias in the same PR.

Two separate prs are expected in that cases. One for breaking changes preview release & one for breaking changes notice release.

Reply:
> > Hi @msJinLei , I want to call out breaking change and remove outdated alias in the same PR.
> 
> Two separate prs are expected in that cases. One for breaking changes preview release & one for breaking changes notice release.

Hi , I have updated the PR, this PR only call out breaking change.